src/analysis/normed_space/basic.lean

```diff
@@ -1186,6 +1187,19 @@ normed_algebra.norm_algebra_map_eq _
 variables (𝕜 : Type*) [normed_field 𝕜]
 variables (𝕜' : Type*) [normed_ring 𝕜']
 
+@[continuity] lemma normed_algebra.algebra_map_continuous
+  [normed_algebra 𝕜 𝕜'] :
+  continuous (algebra_map 𝕜 𝕜') :=
+begin
+  change continuous (algebra_map 𝕜 𝕜').to_add_monoid_hom,
+  exact add_monoid_hom.continuous_of_bound _ 1 (λ x, by simp),
```

Comment:
We have `linear_map.continuous_of_bound` too, it just lives in `normed_space.operator_norm` rather than `normed_space.basic`.  It's slightly aesthetically displeasing either way -- either one has the spurious move to a different category (linear maps vs add_homs) or one has this live in the "wrong" file.  You decide!

Reply:
Added a comment.